ui/templates/data.html

```diff
@@ -11,13 +11,6 @@
 <div id="data">
```

Comment:
Nit: both for the id of this div, and for the name of the include file, "data" is pretty generic. Could this be called "listing-search-results" or similar? Not a blocker; I'm just casting about for suggestions on an otherwise clean, tidy PR.